### PR TITLE
Modify deploy script from git-based to local-file-based style

### PR DESCRIPTION
**Pull request by harrydrippin**
I modified deploy script more conveniently. Now, this script will zip your codes and directly push to the server by `scp`. From now on, you have no needs to commit & push due to upgrade your server code! Yay!

Other files are not special; will be deleted in future.

Please take a look!

### CODE REVIEW

File: deploy/deploy [hunk 1]
```diff
@@ -1,18 +1,22 @@
 #!/bin/bash
 # Deploy Script for Frontend and Backend
-ssh -tt ubuntu@quadcore.news BRANCH=$1 'bash -s' << 'DEPLOY'
+cd ../..
```
Comment (royeom) [inline deploy/deploy:3]: I think ' cd / ' is simpler :)

**Review by royeom — APPROVED** [review]
Cooool~

File: deploy/deploy (hunk 1 above)
Comment (harrydrippin) [reply deploy/deploy:3]: Not intended that! That line moves my working directory from `<PROJROOT>/quadcore-serving/deploy` to `<PROJROOT>`, so it's correct :)

Comment (royeom) [reply deploy/deploy:3]: Good !

**Pull request merged**